configs/swarm/docker-compose.local.yml

```diff
@@ -54,7 +48,7 @@ services:
       "
   irohad1:
     depends_on:
-    - builder
+    - irohad0
```

Comment:
why do other services depend on the 1st?

Reply:
If they don't, they will all build the image redundantly, causing the OOM killer kicking in